packages/vue-inertia/src/types.ts

```diff
@@ -0,0 +1,15 @@
+import { Config, NamedInputEvent, RequestMethod, SimpleValidationErrors, ValidationConfig, ValidationErrors } from 'laravel-precognition'
+import { Form as PrecognitiveForm } from 'laravel-precognition-vue/dist/types'
+import { InertiaForm } from '@inertiajs/vue3'
+
+export type Form<Data extends Record<string, unknown>> = Omit<PrecognitiveForm<Data>, 'setErrors'|'touch'|'forgetError'|'setValidationTimeout'|'submit'|'reset'|'validateFiles'|'setData'|'validate'> & InertiaForm<Data> & {
```

Comment:
@timacdonald my eyes glossed over trying to make sense of this type definition. What do you think of just pulling this into an intermediate type?

```suggestion
type RedefinedProperties = 'setErrors'|'touch'|'forgetError'|'setValidationTimeout'|'submit'|'reset'|'validateFiles'|'setData'|'validate'

export type Form<Data extends Record<string, unknown>> = Omit<PrecognitiveForm<Data>, RedefinedProperties> & InertiaForm<Data> & {
```